go/client/getentries.go

```diff
@@ -28,8 +32,21 @@ func rawGetEntries(ctx context.Context, httpClient *http.Client, logURL string,
 	if end < start {
 		return nil, errors.New("start should be <= end")
 	}
+
+	baseURL, err := url.Parse(strings.TrimRight(logURL, "/") + GetEntriesPath)
+	if err != nil {
+		return nil, err
+	}
+
+	baseURL.RawQuery = url.Values{
+		"start": []string{strconv.FormatInt(start, 10)},
+		"end":   []string{strconv.FormatInt(end, 10)},
+	}.Encode()
+
+	log.Printf("baseURL: %v", baseURL.String())
```

Comment:
Oops, that's actually a log I had in there while debugging, I should have removed it!